src/mixins/focusOnDoubleClick.js

```diff
@@ -0,0 +1,48 @@
+let timeoutID;
+
+function waitToTriggerOpenAnimation() {
+  return new Promise(resolve => {
+    timeoutID = setTimeout(resolve);
+  });
+}
+
+async function focusNameInput(cellView) {
+  if (cellView.model !== this.shape) {
```

Comment:
Works great when I just drag on a few elements, but when I connect to a sequence flow some don't allow doubleclick on the labels anymore. If I add a Text Annotation or End Event after connecting a sequence flow, they also don't seem to allow doubleclick.
![doubleclick-lables](https://user-images.githubusercontent.com/6653340/75193447-4669cf00-5724-11ea-82c9-f6ce359dcbfb.gif)


It'll say:
`cellView.model` is `processmaker.components.nodes.task.Shape` and `this.shape` is `standard.Link`
Or `focusNameInput` stops firing for some elements.

Reply:
If I delete the sequence flow, I can doubleclick the other labels again.

Reply:
I can't seem to reproduce this, even following the exact steps you took in your gif. Do you have a list of steps that I can follow to ensure this error shows up?

Reply:
I can also reproduce this:
 - drag a task (A) to the diagram
 - drag another task (B) to the diagram
 - connect A to B with a sequence flow

I can now double click to edit A but not B.

Deleting the sequence flow makes B editable via double click again.

Reply:
Ah, I can finally reproduce this, but only in Safari (v13); it works without issue for me in Chrome 80 and FF 72 and 73.

@lisa-fehr @johnblackspear what browsers / browser versions did you use to reproduce this?

Reply:
@chipit24 I saw it in Chrome Version 80.0.3987.122